### DISCUSSION TITLE
Number of imported LDAP users

### DISCUSSION THREAD

**Discussion by Dimawesome** [DirectoryTree/LdapRecord#410] [Q&A]
Hello everybody,

I'm trying to import LDAP users, using `php artisan ldap:import users` command. After `directorytree/ldaprecord v2.10.0` update, only 1000 users can be imported, although there are much more of them. As far as I noticed, changes in `src/Query/Pagination/Paginator.php` file led to this result.
So, how could I fix this problem? 

Any help would be appreciated. Thank you.

**Comment by stevebauman** [comment]
Hi @Dimawesome, I've converted this discussion to an issue: https://github.com/DirectoryTree/LdapRecord/issues/411
